distribution/src/conf/log4j2.properties

```diff
@@ -191,7 +192,7 @@ loggers = AUDIT_LOG, SERVICE_LOGGER, trace-messages, org-apache-coyote, com-haze
   BlueprintExtender, ModuleDeployer, VFSTransportSender, PassThroughHttpSender, PassThroughHttpSSLSender, \
   PassThroughHttpListener, PassThroughHttpSSLListener, DeploymentEngine, TaskServiceImpl, SynapseControllerFactory, \
   Axis2SynapseController, MultiXMLConfigurationBuilder, XMLConfigurationBuilder, \
-  VFSTransportListener, NTaskTaskManager, MediationStatisticsComponent, SynapseConfigurationBuilder, ServerManager
+  VFSTransportListener, NTaskTaskManager, MediationStatisticsComponent, SynapseConfigurationBuilder, ServerManager, correlation
```

Comment:
Do we need correlation logs enabled by default? Also, let's add this into the below section with verbose logging aswell?

Reply:
Since we use a system property to enable the co-relation logs, having this by default won't enable it. It is added by default to reduce the config changes user has to do to enable it.

Reply:
Let's add it to the verbose section aswell.